### DISCUSSION TITLE
sktime is now python 3.10 compatible, including the >> developer suite <<

### DISCUSSION THREAD

**Discussion by oo777** [alan-turing-institute/sktime#3012] [Q&A]
I am new to sktime and I was wondering what >> developer suite << is now supported? Thanks!

**Comment by fkiraly** [comment]
That referred to the `sktime` developer tools, including:

* continuous integration https://www.sktime.org/en/stable/developer_guide/add_estimators.html
* tools for creating and validating your own `sktime` compatible estimator https://www.sktime.org/en/stable/developer_guide/add_estimators.html
* the testing framework https://www.sktime.org/en/stable/developer_guide/testing_framework.html

**Reply by oo777** [reply]
Got it, cheers!
